<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LastSubmittal</class>
 <widget class="QMainWindow" name="LastSubmittal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="finalTranslation">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>871</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>译文合并</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Paste">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>330</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:white;}</string>
    </property>
    <property name="text">
     <string>粘贴</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reDo">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>330</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:white;}</string>
    </property>
    <property name="text">
     <string>撤销</string>
    </property>
   </widget>
   <widget class="QPushButton" name="complete">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>500</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>最终提交</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="translatorTable">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>871</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请从上方译者对应框中剪切文本，粘贴到下方提交框中。</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
